docs/user-guide/geo.rst

```diff
@@ -0,0 +1,70 @@
+Geospatial Data
+===============
+
+What geospatial data is supported?
+----------------------------------
+
+In general, the Python Geo Interface is supported; see https://gist.github.com/sgillies/2217756
+
+There are three layers of information:
+
+- The outermost layer is a FeatureCollection
+- Within a FeatureCollection are Features. Each feature has a type, an id, an optional
+  geometry, an optional bbox (bounding box), and optional properties.
+- A geometry has a type and coordinates.
+
+Although the Python Geo Interface is non-prescriptive when it comes to the data types of
+properties, Unify has a more restricted set of supported types. See https://docs.tamr.com/reference#attribute-types
+
+Unify allows any number of geometry attributes per record; the Python Geo Interface is limited to
+one. When converting Unify records to Python Geo Features, the first geometry attribute will
+be used as the geometry; all other geometry attributes will appear as properties.
```

Comment:
I am not used to working with libraries like this, but could this become a problem? We may need to provide some more guidance/examples about how to utilize these properties to accomplish certain goals, since it sounds like there is a slight inequality about how we are using geometries in Unify versus python.

Reply:
I should note that these examples would probably be most at home in the rules for converting section below. You can even anchor link from this spot to a few lines down where you add those examples.

Reply:
It's a good question wether we should convert everything that looks like a geometry.
I expect it to be a fringe case.